notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2261 @@
+{
```

Comment:
<p>Suggestion for bullet 4</p><p> </p><p>The output of the batch prediction is given as input for the <code>ModelEvaluationRegressionOp</code> component. For a custom model, the ground truth cannot be part of the batch prediction instance, so we provide the output of the data sampler with ground truths to <code>ModelEvaluationRegressionOp</code>'s <code>ground_truth_gcs_source</code>  parameter. </p><p> </p><p>For bullet 5:</p><p>Delete "At last"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>

Reply:
<p>done</p>